cpu/kinetis_common/random_rnga.c

```diff
@@ -55,7 +55,7 @@ int random_read(char *buf, unsigned int num)
         }
     }
 
-    return count;
+    return (int)count;
```

Comment:
yes, and it will take a crazy long time to generate that entropy. But I think this is an actual API bug. The documentation for `periph/random.h` says:

>Reads num or less bytes of randomness from the source, will block until random data is available.
>
>Parameters
>    [in]	buf	destination buffer to write the bytes to
>    [in]	num	number of bytes to get from device, only values >0 are valid
>
>Returns
>    the number of bytes written to buf 
>    0 on error 

so I guess we should update either the return type or the docs. My vote is for the return type.